stateproof/builder.go

```diff
@@ -355,50 +387,64 @@ func (spw *Worker) sigExistsInDB(round basics.Round, account basics.Address) (bo
 	return exists, err
 }
 
+func (spw *Worker) builderExistsInDB(rnd basics.Round) (bool, error) {
+	var exist bool
+	err := spw.db.Atomic(func(ctx context.Context, tx *sql.Tx) error {
+		var err2 error
+		exist, err2 = isBuilderExists(tx, rnd)
+		return err2
+	})
+
+	return exist, err
+}
+
 func (spw *Worker) builder(latest basics.Round) {
+	spw.ledger.RegisterVotersCommitListener(spw)
```

Comment:
You are indeed correct. I could modify the voters tracker to keep track of the last round it actually worked on and build builders starting from that round instead of oldDbRound, or we could call this from somewhere else. Any ideas where to call it from?